README.md

```diff
@@ -228,24 +190,11 @@ Don't worry, it's still around. You can install it by running:
 
 ## Contributing
 
-Titanium is an open source project. Titanium wouldn't be where it is now without
-contributions by the community. Please consider forking this repo to improve,
-enhance or fix issues. If you feel like the community will benefit from your
-fork, please open a pull request.
-
-To protect the interests of the Titanium contributors, Appcelerator, customers
-and end users we require contributors to sign a Contributors License Agreement
-(CLA) before we pull the changes into the main repository. Our CLA is simple and
-straightforward - it requires that the contributions you make to any
-Appcelerator open source project are properly licensed and that you have the
-legal authority to make those changes. This helps us significantly reduce future
-legal risk for everyone involved. It is easy, helps everyone, takes only a few
-minutes, and only needs to be completed once.
-
-[You can digitally sign the CLA](http://bit.ly/app_cla) online. Please indicate
-your e-mail address in your first pull request so that we can make sure that
-will locate your CLA. Once you've submitted it, you no longer need to send one
-for subsequent submissions.
+Titanium is an open source project. Titanium wouldn't be where it is now without contributions by the community. Please consider forking this repo to improve, enhance or fix issues. If you feel like the community will benefit from your fork, please open a pull request.
```

Comment:
I would vote to remove the entire Contributing and License section for now, then we can add it back when we sort out the readme boilerplate and copyright stuff.